packages/adminjs-e2e/src/admin.options.js

```diff
@@ -1,4 +1,5 @@
 const { default: AdminJS } = require('adminjs')
+const ComponentLoader = require('adminjs/utils/component-loader')
```

Comment:
Let's just remove the entire `adminjs-e2e` folder since it's never been officially released AFAIK and it's not updated after v4.